packit/local_project.py

```diff
@@ -216,7 +216,7 @@ def _parse_git_project_from_repo_namespace_and_git_project(self,) -> bool:
         return False
 
     def _parse_git_service_from_git_project(self):
-        if self.git_project and not self.git_service and not self.offline:
+        if (self.git_project is not None) and not self.git_service and not self.offline:
```

Comment:
Because of the [lazy-object-proxy](https://pypi.org/project/lazy-object-proxy/). (Testing to `not None` does no need to know the exact value and will not unpack the value from the proxy.)